components/EventItem.tsx

```diff
@@ -0,0 +1,32 @@
+import type { NextPage } from 'next'
+import type { Event } from '../services/GraphQL/types/events'
+import { useRouter } from 'next/router'
+import styles from '../styles/Events.module.css'
+
+const EventItem: NextPage<{ event: Event }> = ({ event }) => {
+  const { push } = useRouter()
+
+  const toEdit = () => {
+    push(`/eventos/editar/${event.id}`)
+  }
+
+  return (
+    <div className={styles['event-item']}>
+      <h2 className={styles['event-header']}>{event.title}</h2>
+
+      <section className={styles['event-content']}>
+        <p className={styles['event-date']}>Fecha: {new Date(event.date).toLocaleDateString()}</p>
```

Comment:
Can you move the date out of the HTML? and then just use the variable

for example: 

```suggestion
        <p className={styles['event-date']}>Fecha: {humanEvent}</p>
```

Reply:
What about `localizedDate` or similar?

Reply:
@anaceciliamolinares any suggestions here?

Reply:
Can you move the date out of the HTML, and then just use the variable?

Reply:
What about  ...  EventDate??